docs/ff-concepts/adding-customization/common-examples.md

```diff
@@ -0,0 +1,284 @@
+---
+slug: /concepts/custom-code/common-examples
+title: Common Examples
+description: Learn about the common custom code examples and use it directly in your project.
+tags: [Custom Actions, Custom Code]
+sidebar_position: 6
+keywords: [FlutterFlow, Custom Actions, Customizations, Flutter, Dart, Pub.dev, Examples]
+---
+
+# Common Code Examples
+
+The custom code feature in FlutterFlow allows you to extend functionality by accessing generated classes and modifying global variables like App States and FlutterFlow themes. This guide covers common scenarios where you can leverage custom code to enhance your project by working directly with data models and other resources within your code.
+
+### Accessing FlutterFlow Generated Classes
+
+Most custom FlutterFlow classes are prefixed with `FF<ClassName>` or `FlutterFlow<ClassName>`. If you need to access these classes in your custom code, simply type "FF" or "FlutterFlow" in the code editor to locate them easily.
+
+![suggestions-dropdown.png](imgs/suggestions-dropdown.png)
+
+### Returning Component from Custom Widget 
+
+In a **[Custom Widget](custom-widgets.md)**, you can integrate a previously built **[FlutterFlow Component](../../resources/ui/components/intro-components.md)** directly, saving you from recreating child content in code. For example, if you’re building a Custom Widget to display custom dialog boxes or bottom sheets using a package from 
```

Comment:
Now that you can pass in components as parameters, you should specify when you would want to pass in vs hardcode the class name. 

I.e. class name may change if you change the name of the component, so may be better to pass in, also need to pass in if dynamic. But if the name wont change and its static, then it can make the code easier and cleaner